ui/app/styles/core/buttons.scss

```diff
@@ -85,13 +128,9 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
       }
 
       &.is-outlined {
```

Comment:
Here is an example is `is-primary.is-outlined` this one is in the disabled state. ![image](https://user-images.githubusercontent.com/6618863/227680345-3c7f6e31-6b97-486b-8279-c37347eb0c48.png)